website/docs/addon_aftereffects_artist.md

```diff
@@ -20,16 +22,44 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 -   [Publish](artist_tools_publisher)
 -   [Manage](artist_tools_inventory)
 
-## Setup
+## AE Extension installation
 
-To install the extension, download, install [Anastasyi's Extension Manager](https://install.anastasiy.com/). Open Anastasyi's Extension Manager and select AfterEffects in menu. Then go to `{path to pype}hosts/aftereffects/api/extension.zxp`. 
+:::tip
+Once your admin has activated `After Effects` in the production bundle, you need to run the AYON launcher at least once. This allows the launcher to download the addon to the correct location on your computer. 
+After that, you can continue with the installation instructions for the After Effects extension.
+:::
+
+<Tabs
+groupId="platforms"
+defaultValue="win"
+values={[
+{label: 'Windows', value: 'win'},
+{label: 'Linux', value: 'linux'},
+{label: 'Mac', value: 'mac'},
+]}>
+
+<TabItem value="win">
+
+1. Use your favorite app to install extensions. we recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).
+2. Locate and install the `extension.zxp` file. You can find it in the `AppData` folder, typically at `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`. Alternatively, you can quickly navigate there using the environment variable shortcut: `%LOCALAPPDATA%\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.
+
+</TabItem>
+
+<TabItem value="linux">
+
+1. Use your favorite app to install extensions.
+2. Locate and install the `extension.zxp` file. You can find it in the user data folder, typically at `~/.local/share/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.
+
+</TabItem>
+
+<TabItem value="mac">
 
-(Current location will be most likely in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\core_X.X.X\ayon_core\hosts\aftereffects\api`.
-On Linux look in `~/.local/share/Ynput/AYON/addons`, on Mac `~/Library/Application Support/Ynput/AYON/addons`)
-This location is temporary until full separation of addons is implemented. Artist must start Tray at least once for Ayon server to populate this location.)
+1. Use your favorite app to install extensions. we recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).
+2. Locate and install the `extension.zxp` file. You can find it in the `Application Support` folder, typically at`~/Library/Application Support/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.
```

Comment:
```suggestion
2. Locate and install the `extension.zxp` file. You can find it in the `Application Support` folder, typically at `~/Library/Application Support/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.
```

Reply:
Good catch. Thank you.
https://github.com/ynput/ayon-documentation/pull/217/commits/5e17173db0cadc8d453509a25a7f6d819216e2a1